packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -0,0 +1,155 @@
+
+## Sign-in flow
+
+Okta supports many ways of authenticating the identity of a user during the sign-in flow. An Okta org administrator creates _policies_, different mixes of authentication methods (also called factors), and assigns them to apps, people, groups, and more. Policies also configure how many authentication methods are required (multifactor), and which methods are optional. This results in a large number of possible combinations for the different factors and steps to complete the sign-in flow.
+
+The Android and iOS Identity Engine SDKs represent the sign-in flow as a state machine. You initialize the machine with the details of your Okta Application Integration, request the initial step in the flow, and cycle through responding to steps until either the user is signed in, cancels, or an error occurs.
+
+<div class="common-image-format">
+
+![A diagram showing the sign-in flow.](/img/mobile-sdk/mobile-idx-basic-flow.png "A diagram that shows the sign-in flow.")
+
+</div>
+
+Each sign-in step may include one or more possible user actions, such as:
+
+- Choosing an authenticator, such as Okta Verify or security questions.
+- Entering a One-Time Passcode (OTP).
+- Cancelling the sign-in flow.
+
+## Sign-in objects
+
+Each step in the sign-in flow is represented by a number of different SDK objects:
+
+<div class="common-image-format">
+
+![A diagram showing the SDK objects and their relationships.](/img/mobile-sdk/mobile-idx-objects.png "A diagram that shows the SDK objects for the sign in flow and the relationships between them.")
+
+</div>
+
+- **Response:** The top-level object that represents a step and contains all the other objects. It also indicates a successful sign-in, and is used to send messages for cancelling the sign-in flow, or to retrieve an access token after the sign-in flow succeeds. A response may contain multiple authenticators and remediations.
+- **Remediation:** Represents the main user actions for a step, such as enrolling in an authenticator or entering an OTP. It's also used to update values entered by the user and to request the next step in the flow.
+- **Authenticator:** Represents a type, or factor for verifying the identity of a user, such as a password or OTP.
+- **Method:** Represents a channel for an authenticator, such as using SMS or voice for an authenticator that uses a phone. An authenticator may have zero or more.
+- **Capability:** A user action, that's associated with a remediation, authenticator, or method, such as requesting a new OTP or a password reset.
+- **Field:** Represents an item displayed in the UI, either a static element, such as a label, or a user input, such as a selection list. It also contains state information, such as whether the associated value is required. Options, or lists of choices, are represented by a collection of fields. A field may also contain a form that contains more fields.
```

Comment:
The first change points out that neither one of us is really satisfied with that wording. Perhaps something like (I've made this change and it will be in the next commit):

> "Represents a UI element, either a static item, such as a label, or user input, such as..."

The second suggestion changes the emphasis and possibly the semantics. In the original "options" and "lists of choices" are the subject, and therefore go up front (active voice.) As the subject, they are represented by collections, rather than collections representing them... collections represent a lot of other things as well.